source-map.bs

```diff
@@ -126,22 +126,22 @@ The document at [https://tc39.es/source-map/](https://tc39.es/source-map/) is th
 up-to-date Source Map specification. It contains the content of the most recently published snapshot
 plus any modifications that will be included in the next snapshot.
 
-If you want to get involved you will find more information under the following [Specification
-Repository](https://github.com/tc39/source-map).
+If you want to get involved you will find more information at the [specification
+repository](https://github.com/tc39/source-map).
 
 <h2 class="no-num">Introduction</h2>
 
 This Ecma standard defines the Source Map format, used for mapping transpiled source code back to the original sources.
 
-The Source Maps format has the following goals:
+The Source Map format has the following goals:
 * Support source-level debugging allowing bidirectional mapping
 * Support server-side stack trace deobfuscation
 
-The original source map format (v1) was created by Joseph Schorr for use by
+The original Source Map format (v1) was created by Joseph Schorr for use by
```

Comment:
The casing seems inconsistent. Here we sentence case but then below it's lower case. I'm not sure if we care?

Reply:
The way I understood it is it upper cases "Source Map format" but for concrete real instances of "source maps" we lower-case it?

Reply:
@nicolo-ribaudo so then in this case, according to your preferred format, we should use "The original source map format"?

Reply:
Yes :)